<!-- You can use a throw statement to raise your built-in exceptions or your
customized exceptions. Later these exceptions can be captured and you can take
an appropriate action. -->

<html>

    <head>
        <script type="text/javascript">
        <!--
            function myFunc() {
                var a = 100;
                var b = 0;
                try {
                    if (b == 0) {
                        throw ("Divide by zero error.");
                    } else {
                        var c = a / b;
                    }
                } catch (e) {
                    document.write("Error: " + e);
                }
            }
        //-->
        </script>
    </head>

    <body>
        <p>Click the following to see the result:</p>
        <form>
            <input type="button" value="Click Me" onclick="myFunc();" />
        </form>
    </body>

</html>